docs/guides/08-introduction-to-sprites.md.WIP

```diff
@@ -0,0 +1,65 @@
+## Introduction to Sprites
+
+CARTO VL has support for sprites, replacing the typical points by images. Those images can be builtin or external and they can also be colored and placed by CARTO VL expressions.
```

Comment:
* can be builtin --> can be **built-in**
* _suggestion_: they can also be colored and placed by CARTO VL expressions. --> they can also be colored and placed by **using** CARTO VL expressions.

Reply:
This is completely WIP, it is not going to be published with the merge, see the file suffix :smile:

Reply:
I know it is a WIP 😂 but that is why there are things that can be fixed during the _WIP_ 😂

Reply:
Well... I'm very open to your commits on this branch :trollface:

Reply:
(except for the one class one file, I'm not open at all to that) :joy: